packages/core/src/indexing-store/sqlite/store.ts

```diff
@@ -338,6 +342,196 @@ export class SqliteIndexingStore implements IndexingStore {
     });
   };
 
+  /*
+   * To determine correct sorting where rows =
+   * [
+   *    id: 0,
+   *    id: 1,
+   *    id: 2,
+   *    id: 3,
+   * ]
+   *
+   * If no after or before arguments, then sort as is;
+   *
+   * If after argument,
+   *   1) sort in same order as passed in
+   *   2) or if no orderBy argument sort as ascending;
+   *
+   * If before argument, then since our query needs to fetch in reverse
+   * (select * from x where id < before) we need to
+   *   1) make orderBy the opposite of the orderBy the user passed in and
+   *      then reverse() the results
+   *   2) or if no orderBy argument, sort in descending;
+   *
+   * after {id : 1} : (select * from x where id > before order by 'asc')
+   * [
+   *    id: 2,
+   *    id: 3,
+   * ]
+   *
+   * before { id: 2} : (select * from x where id < before order by 'desc')
+   * 1)
+   * [
+   *    id: 1,
+   *    id: 0
+   * ]
+   * 2) reverse()
+   * [
+   *    id: 0,
+   *    id: 1
+   * ]
+   */
+
+  findManyPaginated = async ({
```

Comment:
Rather than add a new method, I think we should just change the API of `findMany` to use cursor pagination.